enforcer/enforcer.py

```
@@ -76,33 +94,35 @@ async def on_msg_enforce(self, message: discord.Message, reason: str):
         log_id = await self.config.guild(message.guild).logchannel()
         if log_id:
             log_channel = message.guild.get_channel(log_id)
-            if log_channel:
+            if channel := self._is_valid_channel(log_channel):
```

This shouldn't silently fail - Add logging.